scripts/eurostat/gdp/us_state_gdp_quarterly_import_config.txt

```diff
@@ -0,0 +1,14 @@
+import: {
+  import_name: "eurostatGDPImport3"
+  provenance_url: "https://ec.europa.eu/eurostat/data/database"
+  provenance_description: "Yearly data on GDP in Europe from Eurostat"
+  # Path to the resolved MCF. Can be a repeated field.
+  mcf_url: "gs://resolved_mcf/external_tables/eurostatGDP3/nodes.mcf-00000-of-00001"
+  # First date of imported data.
+  start_date: "2000-01-01"
+  # Last date of imported data.
+  end_date: "2018-12-31"
+  # Source data release schedule.
+  update_frequency: "P1M"
```

Comment:
I am confused by the "P1M" compared with the "yearly" above.  "P1M" also occurs elsewhere in this pull request.

Reply:
It should be P1Y here.

start_date: "2000"
end_date: "2018
update_frequency: "P1Y"

Reply:
Done.